tests/Security/Voter/CitizenProject/CreateCitizenProjectVoterTest.php

```diff
@@ -98,7 +99,7 @@ private function getAdherentMock(bool $isAdherent, bool $isReferent = false, boo
             ->willReturn($isAdherent)
         ;
 
-        $adherent->expects($isReferent ? $this->once() : $this->any())
+        $adherent->expects($this->any())
```

Comment:
This part can be deleted or should say `$this->never()` so you can remove this line https://github.com/EnMarche/en-marche.fr/pull/2596/files#diff-ec0a36def05504ca8eb1a1fb8f06cf34R84.